components/company-name.jsx

```diff
@@ -67,4 +69,5 @@ CompanyName.propTypes = {
 	fieldLabel: PropTypes.string,
 	isRequired: PropTypes.bool,
 	isHidden: PropTypes.bool,
+	maxlength: PropTypes.string,
```

Comment:
Should this be a number?
```suggestion
	maxlength: PropTypes.number,
```